### PR TITLE
chore(vagrant): Switched to ubuntu/xenial64.

### PR DESCRIPTION
**Pull request by robertvolkmann**
Description
---
Alternative approach to #1166. The Vagrant Box `ubuntu/xenial64` installs the necessary Guest Additions for the VirtualBox version of the host. Hence we usage of `rsync` to sync the folder fossology is not necessary.

Changes
---
* Switched to ubuntu/xenial64
* Removed symlinks.
* Added missing test dependency.
* Added script to configure vagrant for development.

How to test
---
```
$ vagrant up
$ vagrant ssh
$ cd /fossology
$ ./utils/prepare-vagrant-dev.sh
$ PGHOST=localhost make test
```

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
 # Copyright Siemens AG, 2014
 # SPDX-License-Identifier:  GPL-2.0 LGPL-2.1
+$post_up_message = <<WELCOME
+Use your FOSSology at http://localhost:8081/repo/
+  user: fossy , password: fossy
+
+Or do a 'vagrant ssh' and look at '/fossology' for your source tree.
+
+Prepare development environment and run tests via:
+  $ cd /fossology
```
[the remark this answers is not in the document]
Comment (maxhbr) [reply Vagrantfile]: I would add the line `   $ vagrant ssh`

[the remark this answers is not in the document]
Comment (robertvolkmann) [reply Vagrantfile]: Done.

**Comment by maxhbr** [comment]
@mcjaeger I think this this PR can be merged

**Comment by mcjaeger** [comment]
indeed it was.

**Pull request merged**